src/components/card/card.js

```diff
@@ -0,0 +1,26 @@
+import React, { PureComponent } from 'react';
+import {
+  Card,
+} from 'reactstrap';
+import propTypes from 'prop-types';
+import './card.scss';
+
+class CardComponent extends PureComponent {
```

Comment:
Figure out how to call this function just `Card` instead of `CardComponent` reach out to Chizzy for help

Reply:
okay, would do that